bindata/network/ovn-kubernetes/004-config.yaml

```diff
@@ -20,3 +20,12 @@ data:
     [gateway]
     mode=local
     nodeport=true
+
+{{ if .OVNHybridOverlayEnable }}
+    [hybridoverlay]
+    enabled=true
+    cluster-subnets="{{.OVNHybridOverlayNetCIDR}}"
+    {{ if .OVNHybridOverlayVXLANPort}}
+    hybrid-overlay-vxlan-port="{{.OVNHybridOverlayVXLANPort}}"
+    {{ end }}
+{{ end  }}
```

Comment:
also, use `{{- if ... }}` and `{{- end }}` everywhere. The `-` after the `{{` makes it discard extra whitespace so that you don't end up with extra blank lines in the generated file (which doesn't _really_ matter, but it looks nicer and it's easy...)

Reply:
I did this except for the first one so that a new line would appear between the sections like we expect